MetaMask/metamask-extension
# Show the Smart Transaction switch for wrapping / unwrapping

## Explanation

Right now we hide Advanced Options on the Build Quote page for wrapping / unwrapping. We did that because a user doesn’t need to set slippage for it, but since we have added the STX switch into Advanced Options, we should only hide slippage instead of the whole Advanced Options section.

## Screenshots/Screencaps

### Before

Advanced Options section was hidden for ETH <-> WETH.

### After

![image](https://user-images.githubusercontent.com/80175477/160402687-34862c47-6f43-4d64-9e84-97c9600df257.png)

## Manual testing steps
- Open Swaps
- Select ETH -> WETH or WETH -> ETH
- Check that the Advanced Options section is visible and has the STX switch in it (for that you would need to set the STX feature flag to true in the `getSmartTransactionsEnabled` function, since it's currently false from our API)
